internal/v1/api.yaml

```diff
@@ -1980,6 +1982,25 @@ components:
           example: [ '---BEGIN CERTIFICATE---\nMIIC0DCCAbigAwIBAgIUI...\n---END CERTIFICATE---' ]
           items:
             type: string
+    AAPCustomization:
+      type: object
+      additionalProperties: false
+      required:
+        - ansible_controller_url
+        - job_template_id
+        - host_config_key
+      properties:
+        ansible_controller_url:
+          type: string
+          example: "example.towerhost.net"
+        job_template_id:
+          type: integer
+          example: 38
+        host_config_key:
+          type: string
+          example: "44d7507f2ead49af5fca80aa18fd24bc"
+        tls_certificate:
+          type: string
```

Comment:
Nitpick: I am assuming this is the AAP CA. While CA essentially is a certificate too, I think this should probably read `tls_certificate_authority` but it is quite long. 🤷🏻